packages/core/contracts-ovm/insured-bridge/implementation/OVM_Testable.sol

```diff
@@ -0,0 +1,31 @@
+// Temp contract until we can use the canonical UMA testable once Optimism support solidity 0.8.
```

Comment:
No modifications right? Worth adding a comment that its copied straight with only the version changed

Reply:
done.